Classes/Service/HubspotFormService.php

```diff
@@ -159,17 +159,19 @@ public function submit(string $formIdentifier, array $formData)
     {
         try {
             $apiResponse = $this->forms->submit($this->settings['api']['portalId'], $formIdentifier, $formData);
-            switch ($apiResponse->getStatusCode()) {
+            switch ($code = $apiResponse->getStatusCode()) {
                 case 204:
-                    return $this->getFormByIdentifier($formIdentifier);
-                case 302:
-                case 500:
+                    return $this->getFormByIdentifier($formIdentifier)['inlineMessage'];
                 default:
+                    $this->logError('Hubspot API returned unexpected code on form submission', $code);
```

Comment:
In this case your method doesn't return anything, but you must return a string.

Reply:
Fixed